[FEATURE] Improve Error Handling for Expect Methods

davidkurilla opened the pull request:
This PR closes Issue #4

I added `match` expressions for the majority of the `.expect` methods.

Three of them included below could not be converted into `match` expression as they do not return a `Result<>` type.

```rs
    let username = toml_data["username"].as_str().unwrap_or_default(); //NO Expects
    let password = toml_data["password"].as_str().unwrap_or_default(); //No Expects
    let dbname = toml_data["dbname"].as_str().unwrap_or_default(); // No Expect
```

We should consider an alternative way to handle these errors in a future issue.

ind1goDusk reviewed (changes requested):
remove println on line 31, otherwise this checks out

JamesM25 commented on src/main.rs:
```diff
@@ -13,23 +13,43 @@ fn main() {
-        .arg(&command)
-        .output()
-        .expect("Failed to execute command");
+    let output = match Command::new("diesel").arg("-c").arg(&command).output() {
```
Is `-c` a valid argument for `diesel`? If not, `.arg("-c")` should be removed.

davidkurilla replied:
I removed the unused .arg(-c) (this was a remnant from when we invoked sh rather than diesel directly)

davidkurilla commented:
addressing both comments:

I removed the debugging `println!` macro
I also removed the unused `.arg(-c)` (this was a remnant from when we invoked `sh` rather than `diesel` directly)

ind1goDusk reviewed (approved):
looks good!
